packages/components/src/LightBox/LightBox.mdx

```diff
@@ -70,3 +70,46 @@ suited by using a dialogue or alternative method.
 ## Props
 
 <Props of={LightBox} />
+
+### onRequestClose
+
+This function receives the index of the last image the user viewed before
+closing the LightBox component with the key lastPosition. This might we useful
+if we want to return information about the final state when closed (e.g. for
+analytics).
+
+<Playground>
+  {() => {
+    const [open, isOpen] = useState(false);
+    return (
+      <>
+        <Button label="😈 Click me!" onClick={() => isOpen(true)} />
+        <LightBox
+          open={open}
+          images={[
+            {
+              title: "😃 Happy Face",
+              url: "https://i.imgur.com/6Jcfgnp.jpg",
```

Comment:
Are these intended to be working URLs? I am not seeing them load when I run this.